template/.zenodo.json.jinja

```diff
@@ -1,14 +1,9 @@
 {
   "creators": [
     {
-      "name": "Castelao, Guilherme",
+    "name": "{{ author_family_name }}, {{ author_first_name }}",
       "affiliation": "National Renewable Energy Laboratory",
-      "orcid": "0000-0002-6765-0708"
-    },
-    {
-      "name": "Pinchuk, Pavlo",
-      "affiliation": "National Renewable Energy Laboratory",
-      "orcid": "0000-0003-4736-4728"
+      "orcid": "{{ orcid }}"
```

Comment:
The placeholder you have in the copier config is the full URL starting with `https`. Is that ok to include here or is it expected that this is the ID only?